bitcoin/wallet.go

```diff
@@ -27,6 +21,14 @@ import (
 	"github.com/btcsuite/btcwallet/wallet/txrules"
 	bip39 "github.com/tyler-smith/go-bip39"
 	"golang.org/x/net/proxy"
+
+	"github.com/OpenBazaar/multiwallet/cache"
+	"github.com/OpenBazaar/multiwallet/client"
+	"github.com/OpenBazaar/multiwallet/config"
+	"github.com/OpenBazaar/multiwallet/keys"
+	"github.com/OpenBazaar/multiwallet/model"
+	"github.com/OpenBazaar/multiwallet/service"
+	"github.com/OpenBazaar/multiwallet/util"
```

Comment:
I'm not sure why the imports are shifting. Can you remove the diff in all places?